src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,45 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import './PostInfo.scss';
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+export const PostInfo = ({ post }) => {
+  const {
+    body,
+    title,
+    user,
+    comments,
+  } = post;
+
+  const NoCommentsMessage = () => (
+    <div className="NoCommentsMessage" data-cy="NoCommentsMessage">
+      <hr />
+
+      <p style={{ fontWeight: 'bold' }}>No comments yet</p>
+    </div>
+  );
```

Comment:
move this into separate component, create folder for messages and collect them in this folder